website/content/api-docs/connect/intentions.mdx

```diff
@@ -48,8 +48,8 @@ The table below shows this endpoint's support for
 <p>
   <sup>1</sup> Intention ACL rules are specified as part of a{' '}
   <code>service</code> rule. See{' '}
-  <a href="/consul/docs/connect/intentions#intention-management-permissions">
-    Intention Management Permissions
+  <a href="/consul/docs/connect/intentions/intentions-usage#acl-requirements">
+    ACL requirements for intentions
   </a>{' '}
   for more details.
 </p>
```

Comment:
can we update this to markdown instead of HTML instead? i don't think we need the <sup>